modules/core/src/main/java/org/apache/ignite/internal/processors/rest/protocols/tcp/GridTcpRestNioListener.java

```diff
@@ -414,7 +411,7 @@ public void onSessionClosed(GridNioSession ses) {
         if (msg instanceof GridClientAuthenticationRequest) {
             GridClientAuthenticationRequest req = (GridClientAuthenticationRequest)msg;
 
-            restReq = new GridRestAuthenticationRequest();
+            restReq = new GridRestTaskRequest();
```

Comment:
Why is this changed like this?

Reply:
I thought that `GridRestAuthenticationRequest` is not needed, but on the other hand I might have gone too far and should revert this particular change.